Makefile.am

```diff
@@ -97,10 +100,13 @@ libriscv__mmachine__@MACHINE_NAME@_a_SOURCES = \
 	src/drivers/sifive,fe310-g000,pll.c \
 	src/drivers/sifive,test0.c \
 	src/drivers/sifive,gpio0.c \
+	src/drivers/sifive,timer.c \
```

Comment:
Currently there is no "sifive,timer" device in the devicetree, and I don't think it makes a lot of sense to add one because it's not an independent hardware block.  IIRC the timers are actually provided by the CLINT, so it should be possible to register the CLINT as a timer driver.